config.json

```
@@ -1802,6 +1802,16 @@
           "if-statements"
         ],
         "difficulty": 5
+      },
+      {
+        "slug": "dot-dsl",
+        "name": "DOT DSL",
+        "uuid": "03e1070d-a3ed-4664-9559-283e535c6bc4",
+        "practices": [],
+        "prerequisites": [
+          "inheritance"
```

We probably should add `lists` as prerequisites as well
```suggestion
          "inheritance",
          "lists"
```

---

I've included this one in the latest commit.